utils/navigator_msgs/srv/ActivationSwitch.srv

```diff
@@ -0,0 +1,8 @@
+# Set to true to request activation of an idling process
```

Comment:
Because in the future things might get a little more complicated and being able to signal if the operation was successful would be useful. It is also something that it seems would be useful for a lot of nodes.